bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,46 @@
+import cowsay
```

Comment:
```suggestion
from cowsay import get_output_string
```
Since you're only using this one function I don't see a reason to import the package as a whole.